libraries/AP_MSP/AP_MSP_Telem_Backend.cpp

```diff
@@ -217,7 +217,7 @@ void AP_MSP_Telem_Backend::update_gps_state(gps_state_t &gps_state)
         gps_state.lon = loc.lng;
         gps_state.alt_m = loc.alt/100; // 1m resolution
         gps_state.speed_cms = gps.ground_speed() * 100;
-        gps_state.ground_course_cd = gps.ground_course_cd();
+        gps_state.ground_course_dd = gps.ground_course_cd() / 10;
```

Comment:
```suggestion
        gps_state.ground_course_dd = gps.ground_course_cd() * 0.1;
```

sorry, its an efficiency thing; multiplication is much faster than division.

Reply:
Two things:
1. This is integer division. Adding a `float` conversion is *not* efficient.
2. If this level of micro-optimisation matters so much, shouldn't we just enable the `-freciprocal-math` compiler optimisation? (Compiler Explorer example: https://godbolt.org/z/5zn7ejnxK)

Reply:
>     1. This is integer division. Adding a `float` conversion is _not_ efficient.

Yep, my bad.  Reviewing too much floating point stuff moving from integer to floating point recently.

>     2. If this level of micro-optimisation matters so much, shouldn't we just enable the `-freciprocal-math` compiler optimisation? (Compiler Explorer example: https://godbolt.org/z/5zn7ejnxK)

I know someone looked at that at some stage but can't remember the resolution.